docker-compose.yml

```diff
@@ -44,6 +44,7 @@ services:
       DATOMIC_ALT_HOST: "acc-text-api"
     volumes:
       - datomic:/srv/datomic/data
+      - ./transactor-logback.xml:/srv/datomic/bin/logback.xml
```

Comment:
can we reuse api/resources/logback.xml instead of creating another config file?

Reply:
well it's different project... of course, random xml file in root dir doesn't look good